README.md

```diff
@@ -50,6 +50,26 @@ python setup.py install
 ```
 
 
+## Docker
+
+Build docker image
+```bash
+docker build -t px-ldap-replace .
+```
+
+Prepare config files
+```bash
+mkdir .config
+vim .config/px-ldap-replace.yaml
```

Comment:
`vim` is a matter of taste. Better use `editor` or `touch` and tell the user to check the "Configuration"-section for values to put in there.